### ISSUE TITLE
Combined NREA/TfL Feed

### ISSUE THREAD

**Issue by rmuggeridge** [peter-mount/departureboards#10]
Hello Peter,

Would be interested to know if it is possible to combine NREA data with that from the TfL API. A number of stations such as Rickmansworth and Amersham only show NR trains. Would be great to see TfL trains combined, especially as they are TfL stations.

**Comment by peter-mount** [comment]
Yes, it's always been on my list to add TfL, it was originally going to be included in departureboards just not had the time to implement it yet.
